src/main/scala/scorex/core/network/peer/PeerManager.scala

```diff
@@ -165,45 +168,35 @@ class PeerManager(settings: ScorexSettings, timeProvider: NetworkTimeProvider) e
 
 object PeerManager {
 
+  trait PeerManagerEvent
```

Comment:
maybe this trait should be moved in the same place where `HandshakedPeer` and `DisconnectedPeer` were moved.

Reply:
I did that at first, but it was also imported by the PeerManager and the statement 
`import NodeViewSynchronizer.ReceivableMessages.PeerManagerEvent`
looked strange, but I can relocate it.

Reply:
I see. Ok.
Could you just add a comment with a "todo" or "fixme", saying that we should reconsider the lcoation of this trait in the future.